ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Protocol/Praos.hs

```diff
@@ -274,12 +360,63 @@ data instance ConsensusConfig (Praos c) = PraosConfig
   }
   deriving (Generic)
 
+instance PraosCrypto c => NoThunks (ConsensusConfig (Praos c))
+  -- use generic instance
```

Comment:
I would prefer to not add comments like `-- use generic instance`. It's a slippery slope to try to explain Haskell semantics in the code's comments. I think `DeriveAnyClass` et al is something we should assume our audience understands.

Reply:
This comment was already there, I just moved the instance, but I agree we can remove those.